### PR TITLE
Feature/135/profile view page

### PR DESCRIPTION
**Pull request by LindaNayeli104**
# What does this PR do?

Developed the user profile component, where the user can see their information and past projects. This information will come from GitHub.

Made the call to get the information from GitHub, built a table to store the information obtained, stored the information in the table, queried and displayed the information on the frontend.

# Where should the reviewer start?

Run the project locally and select the profile option by clicking your email in the home page navigation bar.

# How should this be manually tested?

- Go to the profile page and check if your GitHub information is displayed in a responsive way
	- Username
	- Projects

# What are the relevant tickets?

135, 150 and 153

# Screenshots

image

image

image

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Review by leslye-patino — APPROVED** [review]
LGTM!

**Review by leslye-patino — APPROVED** [review]
LGTM!

**Review by martinrobled0 — CHANGES_REQUESTED** [review]

File: app/auth.server.ts [hunk 1]
```diff
@@ -68,7 +71,23 @@ let auth0Strategy = new Auth0Strategy(
           benchStatus: lakeProfile.contact__status,
         });
       }
-      // Get the user data from your DB or API using the tokens and profile
+      const userGitHubProfile = await getGitHubProfileByEmail(email);
+      if (userGitHubProfile?.email === '' || userGitHubProfile?.email === null || userGitHubProfile?.email === undefined)  {
+        const { data: userInfo } = await getUserInfo(email);
+        const { data: repos } = await getUserRepos(userInfo.items[0].login)
+        createGitHubProfile(email, userInfo.items[0].login, userInfo.items[0].avatar_url, userInfo.items[0].repos_url);
```
[the remark this answers is not in the document]
Comment (LindaNayeli104) [reply app/auth.server.ts]: Done!

**Review by martinrobled0 — APPROVED** [review]
good job! lgtm

**Pull request merged**